.circleci/config.yml

```diff
@@ -41,3 +46,48 @@ jobs:
             test/examples/run.sh
       - store_artifacts:
           path: coverage
+  api-governance:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/enigma.js
+      - run:
+          name: Create version.txt
```

Comment:
a little bit overkill for getting the version.
you can a `VER=$(cat package.json | jq .version)` instead